safir/src/safir/metrics/_config.py

```diff
@@ -38,45 +82,87 @@ class MetricsConfiguration(BaseSettings):
         validation_alias=AliasChoices("appName", "METRICS_APP_NAME"),
     )
 
-    disable: bool = Field(
-        default=False,
-        title="Disable",
-        description='Set to "True" to prevent actually publishing metrics',
+    events: EventsConfiguration = Field(
+        default_factory=EventsConfiguration,
+        title="Events configuration",
     )
 
     model_config = SettingsConfigDict(
-        case_sensitive=False,
-        env_prefix="METRICS_",
+        alias_generator=to_camel,
```

Comment:
This doesn't conflict with the [validation_alias](https://github.com/lsst-sqre/safir/pull/311/files#diff-5c497f6272460b4ab381bc8d81a2801f0dc6c6fd90f945257c889e77fac4c569R82) declared on the `app_name` field?

Reply:
No, alias generators are ignored if `validation_alias` is explicitly defined. This area of Pydantic settings is kind of confusing.